scripts/words-scraper/id.js

```diff
@@ -0,0 +1,115 @@
+const puppeteer = require("puppeteer");
+const cheerio = require("cheerio");
+const fs = require("fs/promises");
+
+const NOUNS_BASE_URL = "https://kbbi.kata.web.id/kelas-kata/kata-benda";
+const ADJECTIVES_BASE_URL = "https://kbbi.kata.web.id/kelas-kata/kata-sifat";
+
+(async () => {
+  const browser = await puppeteer.launch();
+  const page = await browser.newPage();
+
+  // Scrape nouns.
+  // `Set` is used because it guarantees uniqueness (whereas a normal Array doesn't).
+  const nouns = new Set();
+  await page.goto(NOUNS_BASE_URL);
+
+  const lastNounPageNumber = await getNumberOfLastPage(page);
+  const nounPageNumbers = getFetchedPageNumbers(lastNounPageNumber);
+  console.log(lastNounPageNumber, nounPageNumbers);
+  for (const pageNumber of nounPageNumbers) {
+    info(`noun page ${pageNumber}`);
+
+    await page.goto(`${NOUNS_BASE_URL}/page/${pageNumber}`);
+    const pageNouns = await getWordsFromPage(page);
+
+    for (const word of pageNouns) {
+      nouns.add(word);
+    }
+  }
+
+  // Scrape adjectives.
+  // `Set` is used because it guarantees uniqueness (whereas a normal Array doesn't).
+  const adjectives = new Set();
+  await page.goto(NOUNS_BASE_URL);
+
+  const lastAdjectivePageNumber = await getNumberOfLastPage(page);
+  const adjectivePageNumbers = getFetchedPageNumbers(lastAdjectivePageNumber);
+
+  for (const pageNumber of adjectivePageNumbers) {
+    info(`adjective page ${pageNumber}`);
+
+    await page.goto(`${ADJECTIVES_BASE_URL}/page/${pageNumber}`);
+    const pageAdjectives = await getWordsFromPage(page);
+
+    for (const word of pageAdjectives) {
+      adjectives.add(word);
+    }
+  }
+
+  console.log({
+    nouns: Array.from(nouns),
+    adjectives: Array.from(adjectives),
+  });
+
+  await browser.close();
+})();
```

Comment:
@ans-4175 usually I put the "main" code above so that I don't have to scroll to find it... I'm happy to relocate it to the bottom of the file. Let me know what you think. Thanks!